org.scala-ide.sdt.core/src/org/scalaide/core/internal/extensions/AutoEdits.scala

```diff
@@ -30,7 +30,8 @@ object AutoEdits {
     fullyQualifiedName[SurroundSelectionWithString] → SurroundSelectionWithStringSetting,
     fullyQualifiedName[SurroundSelectionWithParentheses] → SurroundSelectionWithParenthesesSetting,
     fullyQualifiedName[SurroundSelectionWithBraces] → SurroundSelectionWithBracesSetting,
-    fullyQualifiedName[SurroundSelectionWithBrackets] → SurroundSelectionWithBracketsSetting
+    fullyQualifiedName[SurroundSelectionWithBrackets] → SurroundSelectionWithBracketsSetting,
+    fullyQualifiedName[SurroundSelectionWithAngleBrackets] → SurroundSelectionWithAngleBracketsSetting
```

Comment:
Yes, the partitions are another problem domain. Don't know yet what is best to solve it. Everywhere vs nowhere sounds like a good solution when we don't want to force users to configure every partition.